src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,47 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ELEMENTS_NUMBER = 10;
+    private final Element<K, V>[] elements = new Element[MAX_ELEMENTS_NUMBER];
+    private int elementCounter = 0;
```

Comment:
what about simple `size`?
```suggestion
    private int size = 0;
```